3.3/.sti/bin/assemble

```diff
@@ -1,23 +1,66 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
+  # For SCL enablement
+  source .bashrc
 
-echo "---> Copying application source ..."
-cp -Rf /tmp/src/. ./
+  set -e
 
-if [ -f setup.py ]; then
-    echo "---> Installing application ..."
-    python setup.py install --user
-elif [ -f requirements.txt ]; then
-    echo "---> Installing dependencies ..."
-    pip install --user -r requirements.txt
-fi
+  echo "---> Copying application source ..."
+  cp -Rf /tmp/src/* ./
 
-# set permissions for any installed artifacts
-chmod -R og+rwx /opt/openshift
+  if [ -f requirements.txt ]; then
+      echo "---> Installing dependencies ..."
+      pip install --user -r requirements.txt
+  fi
 
-# remove pip temporary directory
-rm -rf /tmp/pip_build_default
+  if [ -f setup.py ]; then
+      echo "---> Installing application ..."
+      python setup.py install --user
+  fi
+
+  # set permissions for any installed artifacts
+  chmod -R og+rwx /opt/openshift
+
+  set +e
+  django_collectstatic
+  set -e
+
+  # remove pip temporary directory
+  rm -rf /tmp/pip_build_default
+}
+
+function django_collectstatic() {
+  [ -n "$DISABLE_COLLECTSTATIC" ] && return
```

Comment:
If you're introducing new variable, please update README.md and don't forget to update openshift-docs as well. We want to have all those places up to date.